fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/rdf/JcrRdfTools.java

```diff
@@ -405,7 +406,8 @@ private Resource getSkolemizedResource(final IdentifierConverter<Resource, Fedor
         final AnonId id = resource.asResource().getId();
 
         if (!skolemizedBnodeMap.containsKey(id)) {
-            final Node orCreateNode = jcrTools.findOrCreateNode(session, skolemizedId());
+            final String path = "/.well-known/genid/" + pidMinter.mintPid();
+            final Node orCreateNode = jcrTools.findOrCreateNode(session, path);
```

Comment:
Ensure that interim path elements are marked with mixin type: FEDORA_PAIRTREE.
See: https://github.com/fcrepo4/fcrepo4/blob/master/fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/services/AbstractService.java#L67